scripts/rollup.config.js

```diff
@@ -1,49 +1,140 @@
 import nodeResolve from "rollup-plugin-node-resolve";
 import commonjs from "rollup-plugin-commonjs";
 import babel from "rollup-plugin-babel";
+import minify from 'rollup-plugin-babel-minify';
+import replace from 'rollup-plugin-replace';
+import svg from 'rollup-plugin-svg'
```

Comment:
This plugin allows to include SVG in the JS library.

Reply:
Yes 👍